analysis/analysis-api-fir/src/org/jetbrains/kotlin/analysis/api/fir/types/KtFirFunctionalType.kt

```diff
@@ -24,6 +24,7 @@ import org.jetbrains.kotlin.analysis.api.types.KtType
 import org.jetbrains.kotlin.analysis.api.types.KtTypeNullability
 import org.jetbrains.kotlin.analysis.low.level.api.fir.util.errorWithFirSpecificEntries
 import org.jetbrains.kotlin.builtins.functions.FunctionTypeKind
+import org.jetbrains.kotlin.fir.resolve.fullyExpandedType
```

Comment:
Not used? I guess you decided to use `valueParameterTypesIncludingReceiver` which actually uses `fullyExpandedType`.

Reply:
Updated.